test/espnet2/asr/frontend/test_s3prl.py

```diff
@@ -37,14 +37,15 @@ def test_frontend_output_size():
 
 
 @pytest.mark.parametrize(
-    "fs, frontend_conf, multilayer_feature",
+    "fs, frontend_conf, multilayer_feature, layer",
     [
-        (16000, dict(upstream="mel"), True),
-        (16000, dict(upstream="mel"), False),
-        (16000, dict(upstream="mel", tile_factor=1), False),
+        (16000, dict(upstream="mel"), True, -1),
+        (16000, dict(upstream="mel"), False, -1),
+        (16000, dict(upstream="mel", tile_factor=1), False, -1),
+        (16000, dict(upstream="mel"), False, 0),
     ],
 )
-def test_frontend_backward(fs, frontend_conf, multilayer_feature):
+def test_frontend_backward(fs, frontend_conf, multilayer_feature, layer):
     if not is_torch_1_8_plus:
         return
 
```

Comment:
It seems that layer is not passed to S3prlFronted

Reply:
can you explain it a bit in details? Not quite get what is your question

Reply:
The argument "layer" is not used in frontend = S3prlFrontend(...)